### PR TITLE
Milestone 3: Unit testing add remove

### PR DESCRIPTION
**Pull request by yetemegn-telaye**
**Todo-List-App milestone 3: Unit testing add remove**

In this milestone we:

- Created a test for adding a todo
- Created a test for removing a todo

### CODE REVIEW

File: src/modules/Crud.test.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  test('remove task from the local storage and li from the DOM', () => {
+    const todoId = 0;
+    const listContainer = document.createElement('li');
+    listContainer.className = 'task-item';
+    listContainer.id = 0;
+    removeTodo(todoId, listContainer, arrTest);
+    expect(localStorage.getItem('tasks')).toEqual('[]');
+    expect(document.contains(listContainer)).toBe(false);
+  });
+});
```
Comment (t-yanick) [inline src/modules/Crud.test.js:17]: - [ ] According to the project [requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/tests_2.md), you are supposed to add test for the following, 
   - a function for editing the task description.
   - a function for updating an item's 'completed' status.
   - the "Clear all completed" function.

Please kindly add them.

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @yetemegn-telaye, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/Crud.test.js (hunk 1 above)
Comment (yetemegn-telaye) [reply src/modules/Crud.test.js:17]: Hey @t-yanick, Thank you so much for telling us that. Yes, we were supposed to submit this pull request for the first project. Can we still submit it for the first project?

**Review by franclobo — COMMENTED** [review]
Hi @yetemegn-telaye, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have the linters and has conflicts that must be solved.

image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by yetemegn-telaye** [comment]
Hey @franclobo , does that mean I should another pull request then?

**Comment by franclobo** [comment]
> Hey @franclobo , does that mean I should another pull request then?

Hi,  @yetemegn-telaye. Yes, you can. First, all the linters must pass and try to solve the merge conflicts, you can review this [video](https://www.youtube.com/watch?v=xNVM5UxlFSA&t=149s)

**Comment by yetemegn-telaye** [comment]
@franclobo, yes I did try to solve the merge request like that but it would not save my changes unless I merge it. But now that I can use a different pull request, what if I also use another branch so that I can base it on my updated master?

**Comment by franclobo** [comment]
@yetemegn-telaye no you don't need to merge it after solving the conflicts and you can´t create another PR. Let's do a Zoom call.

**Review by Santiago220991 — APPROVED** [review]
Hi @yetemegn-telaye @letarulinda , 
I'm Santiago your reviewer for this PR. :wave: :wave: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
:heavy_check_mark: Applied changes suggested by the last reviewer.
:heavy_check_mark: Followed GitHub flow.
:heavy_check_mark: No linter errors.

## Optional suggestions
[OPTIONAL] Kindly make sure that the user can see the new task added without the need to refresh the page. This is going to help to improve the user's experience. :+1: :smiley: :smiley: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**